### PR TITLE
Blog App - Add forms

### PR DESCRIPTION
**Pull request by anthonymr**
## Milestone 7: Add forms
In this PR, I completed the following steps:

-  Create a method called `current_user` in your ApplicationController to make current user data available to all controllers.
    -  It returns the first user from the database.
- Create forms to perform the following functions:
    - Creates a Post on behalf of the `current_user`.
    - Create a comment on behalf of the `current_user`.
- Allow Users to add likes to Posts.

ℹ️ To run this project:

- Clone this repository to your local machine:

```sh

  git clone git@github.com:anthonymr/anthony-blog.git
  cd anthony-blog
```
- Install dependencies:
```sh
  bundle install
```
- Recreate Data Base:
```sh
  rake db:schema:load
```
Run `Rails` server:

```  sh
  rails s  
```

### CODE REVIEW

File: app/controllers/posts_controller.rb [hunk 1]
```diff
@@ -8,4 +8,35 @@ def show
...
+    end
+  end
+
+  def add_like
+    @post = Post.find(params[:id])
+    Like.create(post: @post, author: current_user)
+    Like.update_like_counter(@post)
+    redirect_to user_posts_path(@post.author)
+  end
+
```
Comment (julie-ify) [inline app/controllers/posts_controller.rb]: - [ ] This like implementation should be in its own Like controller, not the Post controller. Kindly create a like controller and move this to that file under the `create` action.

**Review by julie-ify — CHANGES_REQUESTED** [review]
## Hi @anthonymr,

Good job so far!

## Highlights!
- Good code organization and working app ✔️
- Descriptive README ✔️ 

Hello ✋, you have done well reaching this point of the milestone 🤝 . But there are some issues that you still need to work on to go to the next project but you are almost there!

# Status: Required Changes ♻️⛔️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread and tag me `@julie-ify` if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app/controllers/posts_controller.rb (hunk 1 above)
Comment (anthonymr) [reply app/controllers/posts_controller.rb]: Hi @julie-ify, thanks for your review, that make sense.

**Review by levy002 — APPROVED** [review]
## Hello @anthonymr 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Status: APPROVED :fire:  :fire: 

## Highlights
- Implemented all suggested changes :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**